src/main/java/org/fundacionjala/at15/katas/bankocr/vladimir/AccesoDatosImpl.java

```diff
@@ -0,0 +1,27 @@
+package org.fundacionjala.at15.katas.bankocr.vladimir;
+
+import java.io.*;
+
+public class AccesoDatosImpl {
```

Comment:
"Impl" sometimes could be redundant for names of classes, if possible, try to avoid it

Reply:
ok Jose